charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -44,6 +44,10 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        {{- if .Values.env }}
```

Comment:
please use `{{- with .Values.env }}`

Reply:
Changed. I preferred to stick with the conditional as it is quite common in this chart.